mucommander-core/src/main/java/com/mucommander/ui/action/impl/ReportBugAction.java

```diff
@@ -31,12 +37,55 @@
  */
 public class ReportBugAction extends OpenURLInBrowserAction {
 
+    private static final String NEW_BUG_FORMAT = "%s?" +
+            "labels=bug&" +
+            "template=bug_report.yml&" +
+            "title=[Bug]%20&" +
+            "version=%s&" +
+            "java-version=%s&" +
+            "os-version=%s&" +
+            "logs=%s";
+
     public ReportBugAction(MainFrame mainFrame, Map<String,Object> properties) {
         super(mainFrame, properties);
+    }
 
-        putValue(URL_PROPERTY_KEY, com.mucommander.RuntimeConstants.REPORT_BUG_URL);
+    @Override
+    public void performAction() {
+        Object url = com.mucommander.RuntimeConstants.REPORT_BUG_URL;
+        try {
+            // TODO remove - temporarily left for testing: uncomment to check on that test-repo
+            //url = "https://github.com/pskowronek/parametrized_issue_template/issues/new";
+            String ver = String.format(Activator.portable ? "%s (%s)" : "%s", RuntimeConstants.VERSION, Translator.get("portable"));
+            String java_ver = String.join(System.lineSeparator(),
+                    System.getProperty("java.version"),

```

Comment:
And a common place to generate that info should be created to be also used by AboutDialog

Reply:
yeah, there's a room for refactoring there, maybe to use `JavaVersion#getRawSystemProperty`

Reply:
Ok, I will take a look

Reply:
I've taken a look - I'm not convinced to have any common place for those versions. I would leave it as is.